source/components/molecules/Banner/Banner.stories.js

```diff
@@ -4,23 +4,35 @@ import StoryWrapper from '../StoryWrapper';
 import Banner from './Banner';
 
 const ILLU_EXPENSES = require('source/assets/images/illustrations/illu_utgifter_margins_1x.png');
-const ICON_INCOME = require('source/assets/images/icons/icn_inkomster_1x.png');
-const ICON_EKB = require('source/assets/images/icons/icn_Main_ekonomiskt-bistand_1x.png');
-const ICON_EXPENSES = require('source/assets/images/icons/icn_utgifter_1x.png');
 
 storiesOf('Banner', module)
-  .add('Icon', props => (
+  .add('Image', props => (
     <StoryWrapper {...props}>
-      <Banner iconSrc={ICON_EKB} />
+      <Banner imageSrc={ILLU_EXPENSES} />
```

Comment:
Is this component locked to EKB or can I use it in any future app.
Maybe the constant ILLU_EXPENSES is not so general?

Reply:
It's just for the story, you can pass any illustration to the component :)

Reply:
So the component takes a image src which is the path to the img that you want to display